docs/AdvancedConfiguration.md

```diff
@@ -287,3 +287,36 @@ We can empty the staging area with a given write by setting the staging size to
 ```scala
 --conf spark.qbeast.index.stagingSizeInBytes=0
 ```
+
+## Pre-commit Hooks
```

Comment:
I think would be useful to include this section in the `QbeastFormat.md` too

Reply:
I'd say they aren't exactly format-related.

Reply:
Yes, but it can affect the commited metadata, right?

Reply:
Yes, the `CommitInfo.` But it won't change any `metadata`.

Reply:
Added example hook output for `CommitInfo.tags.`